plugins/Marketplace/stylesheets/plugin-details.less

```diff
@@ -149,6 +149,13 @@
         top: 5% !important; // required to override inline style set by the modal
     }
 
+    @media (max-width: 480px) {
+        // full screen takeover
+        min-width: 100vw;
```

Comment:
```suggestion
        width: 100%;
```

Setting `100vw` could behave rather odd for some browsers.

For desktop scrollbars visible in some browsers it will look wrong with `100vw`, and probably even worse with a suitable `100%` replacement:

![](https://github.com/matomo-org/matomo/assets/635801/b2e0f5bf-3308-43b9-9d8a-7032c82b3ac0)
_100vw moves content below the parent page scrollbar, hiding it in my Linux Chrome while still working_

![](https://github.com/matomo-org/matomo/assets/635801/5aaedbe5-6df9-4e96-a451-a867d0ab6080)
_100% displays two scrollbars for modal and parent page_

Scrollbars are from a Linux Desktop Chrome without device emulation.

Firefox has it's own set of problems with the scrollbar hidden by default, and it seems to always select the background page one when moving the mouse pointer over it, the modal scrollbar is always hidden behind it in an unusable state. And once scrolling to the end of the modal content the background automatically begins to scroll.

So the suggestion is more a "optional: we could consistent definitions/units across breakpoints", not "here is something to fix".

I would not expect any user to find this oddity and then go forward to report is as a problem :see_no_evil:

Reply:
@mneudert I've added a tweak that should prevent the scroll of the underlying page when the modal is open. It also adds a bit of margin when scrolling is disabled so that the page doesn't jump under the modal.
It should prevent the dual scroll bars and any other oddities. Can you please test it out?

Reply:
It works, but I would say it is probably not worth the effort.

Eventually we would need a platform wide decision to either allow or disallow background scrolling once a modal is open. And then have that solved by a global modal wrapper to not implement it over and over. Right now we would create a single page that is in itself not consistent with the behaviour.

The fake scrollbar space itself creates a new alignment issue for the whole `90%` width breakpoint range:

![](https://github.com/matomo-org/matomo/assets/635801/d90f72f0-5e31-4359-9163-407145e09889)

So unless it is a UX requirement to prevent both the double scrollbars and disable the body scrolling, I would either completely ignore that edge case or switch from `100vw` to `100%` and then ignore it.

Reply:
Ok, fair enough. I'll revert the addition and create a new internal ticket for it to address it separately.

Reply:
I think the alignment on the 90% is correct, even when it doesn't look so. The main point is that the page doesn't jump under the modal, so the scrollbar is replaced with margin. The plugin card would still sit where it sits even with the scrollbar.